src/imp_pl.rs

```diff
@@ -86,6 +86,10 @@ impl<T> OnceCell<T> {
         res
     }
 
+    pub(crate) fn wait(&self) {
+        unimplemented!()
+    }
```

Comment:
Does anyone know if it's actually possible to implement this with parking lot without incerasing the size_of onec_cell?